src/components/VideoMenuOptions/VideoMenuOptions.jsx

```diff
@@ -14,32 +18,36 @@ const VideoMenuOptions = (props) => {
   const menuOptionsRef = useRef();
   useOnClickOutside(() => setVideoMenuOptionsView(false), menuOptionsRef);
   return (
-    <div ref={menuOptionsRef} className={`relative-position`}>
-      <button
-        className={`${styles.video_menu_btn} g-flex-row g-flex-center`}
-        onClick={toggleMenuOptionsView}
-      >
-        <span className="material-icons">more_vert</span>
-      </button>
-      {videoMenuOptionsView ? (
-        <div className={`${styles.menuOptionsContainer}`}>
-          <ul className={`${styles.menuOptionsList} g-flex-column`}>
-            <li
-              className={`${styles.menuOptionsListItem} g-flex-row g-flex-align-center`}
-            >
-              <span className="material-icons-outlined">watch_later</span>
-              REMOVE FROM WATCH LATER
-            </li>
-            <li
-              className={`${styles.menuOptionsListItem} g-flex-row g-flex-align-center`}
-            >
-              <span className="material-icons-outlined">thumb_up</span>
-              ADD TO LIKED VIDEOS
-            </li>
-          </ul>
+    <>
+      {isUserLoggedIn ? (
```

Comment:
you can use `&&` operator here

Reply:
Yes the thought crossed my mind of using && operator too!! But then got to know that using ternaries was better.
Followed this blog: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx